samples/system_time/Makefile

```diff
@@ -0,0 +1,4 @@
+BOARD ?= frdm_k64f
```

Comment:
"Test gettimeofday() and settimeofday()."

Since this is a build_only sample and not a proper test, we really aren't testing anything.
Can you rewrite this as a proper test case which can be tested in the QEMU emulators, or at least on some hardware platform in our test case framework? Surely there must be some way we can programmatically prove that this mechanism works.

Reply:
Perhaps settimeofday, wait n seconds, and see if gettimeofday is n seconds later, where n isn't a value too large to hold up testing -- maybe 5-10 seconds?